scripts/deploy.sh

```diff
@@ -7,7 +7,7 @@ APP='janis-staging'
 CURRENT_CONFIG=$(heroku config --app "$APP")
 
 if [[ $CURRENT_CONFIG != *"REACT_APP_CMS_ENDPOINT"* ]]; then
-    heroku config:set REACT_APP_CMS_ENDPOINT=https://joplin-staging.herokuapp.com/api --app "$APP"
+    heroku config:set REACT_APP_CMS_ENDPOINT=https://joplin-staging.herokuapp.com/api REACT_APP_CMS_ASSETS=https://joplin-staging.herokuapp.com/media --app "$APP"
```

Comment:
@benweatherman can I get your 👀 on this change to the deploy script?

Reply:
👍 I've also added it to the heroku config at https://dashboard.heroku.com/apps/janis-staging/settings


![config](https://user-images.githubusercontent.com/272675/35164736-3aabd966-fd11-11e7-992b-6201e33d2075.png)

Reply:
Actually, now that I'm looking at this, we should probably set this in a different if statement like such

```sh
 if [[ $CURRENT_CONFIG != *"REACT_APP_CMS_ASSETS"* ]]; then
  heroku config:set REACT_APP_CMS_ASSETS=https://joplin-staging.herokuapp.com/media --app "$APP"
fi
```

This will ensure that if the variable isn't in the config, it gets set